### PR TITLE
Added 2 folders: contains, pages. And changed 5 tests.

### PR DESCRIPTION
**Pull request by 24alinka**
Hello. Could you help me with this task? Because I'm a little confused. And I'm having trouble with some tests. Please check and let me know what I need to change to make my tests work. Thank you!

### CODE REVIEW

File: test_start_page1.py [hunk 1]
```diff
@@ -1,205 +1,80 @@
...
+        self.log.info("Signed Up as user %s", email_value)
+        # Verify error message
+        start_page.verify_email_error(email_value)
+        self.log.info("You must provide a valid email address")
+
+    def test_same_email(self, start_page):
+        # Prepare data
+        user = random_str()
+        username_value = f"{user}{random_num()}"
+        email_value = start_page.error_email("testalinka-tets@gmail.com")
```
Comment (QALightLecorPython) [inline test_start_page1.py]: What do you expecting from this step ? It returns None and fail your test

File: test_start_page1.py [hunk 2]
```diff
@@ -1,205 +1,80 @@
...
+        start_page.verify_name_error()
+        self.log.info("Error was verified")
+
+    def test_incorrect_password(self, start_page):
+        # Fill Password
+        start_page.error_password("test")
+        self.log.info("Please enter a valid password")
+
+        # Verify password error
+        start_page.verify_password_error()
```
Comment (QALightLecorPython) [inline test_start_page1.py:44]: Please add some sleep before verification

File: test_start_page1.py [hunk 3]
```diff
@@ -1,205 +1,80 @@
...
+        username_value = f"{user}{random_num()}"
+        email_value = start_page.error_email("testalinka-tets@gmail.com")
+        password_value = f"{random_str(6)}{random_num()}"
+
+        # Sign Up as a user
+        start_page.sign_up(username_value, email_value, password_value)
+        self.log.info("Signed Up as user %s", username_value)
+
+        # Verify error message
+        start_page.verify_email_error(email_value)
```
Comment (QALightLecorPython) [inline test_start_page1.py]: This method does not expect any arguments

**Review by QALightLecorPython — CHANGES_REQUESTED** [review]

File: pages/start_page.py [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        self.fill_field(xpath=self.constants.SIGN_UP_USERNAME_FIELD_XPATH, value=username)
+
+    def error_password(self, password):
+        """Sign Up with invalid password"""
+        self.fill_field(xpath=self.constants.SIGN_UP_PASSWORD_FIELD_XPATH, value=password)
+
+    def error_email(self, email):
+        """Sign up twice with the same email"""
+        self.fill_field(xpath=self.constants.SIGN_UP_EMAIL_FIELD_XPATH, value=email)
+
```
Comment (QALightLecorPython) [inline pages/start_page.py]: These methods are useless. Please reuse sign_up method with empty fields for such purpose

**Review by QALightLecorPython — CHANGES_REQUESTED** [review]

**Comment by 24alinka** [comment]
Hi again, I fixed my tests, but I still have a problem with one test (def test_invalid_email). Could you check again and help me, please?

**Review by QALightLecorPython — APPROVED** [review]

**Pull request merged**